fixbackend/dispatcher/dispatcher_service.py

```diff
@@ -190,12 +206,30 @@ async def collect_job_finished(self, message: Json) -> None:
             )
             for account_id, account_details in account_info.items()
         ]
-        collected_accounts = [CloudAccountId(UUID(account_id)) for account_id in account_info.keys()]
+        # lookup the cloud account id from the job_id
+        cloud_account_id_bytes: Optional[bytes] = await self.temp_store_redis.hget(
+            self._jobs_hash_key(workspace_id), job_id
+        )  # type: ignore
+        if cloud_account_id_bytes is None:
+            log.error(f"Could not find cloud account id for job id {job_id}")
+            return
+        cloud_account_id = cloud_account_id_bytes.decode()
+
+        account_progress_bytes: Optional[bytes] = await self.temp_store_redis.hget(
+            self._collect_progress_hash_key(workspace_id), cloud_account_id
+        )  # type: ignore
```

Comment:
redis should give you strings not bytes - or?

Reply:
But it’s not.